packages/gatsby-source-contentful/fetch-data.js

```diff
@@ -68,6 +68,7 @@ module.exports = function () {
             // someone develop a contentful site even if not connected to the internet.
             // For prod builds though always fail if we can't get the latest data.
             process.exit(1);
+            break;
```

Comment:
This line would never be executed since process exited before, right ?

Reply:
Not really. Our lint check told us that there should be always a break or return.

Reply:
@DanielRuf @ZYSzys is right, when running process.exit the process stops which means break isn't needed. It might be best practise to be consistent with breaks. A better way is to use process.exitCode = 1; and let the node process gracefully stop but atm this isn't possible with the current setup.